### ISSUE TITLE
Option for search shortcut

### ISSUE THREAD

**Issue by agg23** [and3rson/clay#47]
Currently the interface essentially forces one to use modifier keys to access most of its functionality, whether that is playing/pausing, adding to the queue, or switching tabs. Interestingly enough, if you bind those functions to single keys, the operations still work, but you can no longer type in a search box.

Ideally this would be toggleable functionality, so if someone binds search to a shortcut (say "s"), then they can use any combination of keys to access other systems, and still be able to type in the search field when "s" is pressed.

**Comment by ValentijnvdBeek** [comment]
I agree that this is a problem especially seeing bind a common modifier + keys (e.x. C-c and C-m) to various signals to be backwards compatible with very old systems. This will randomly break people's configurations because it will suddenly not work without any explanation. 

So I hundred percent agree and, funnily enough, started working on this just today.

**Comment by agg23** [comment]
I'm glad I created an issue then, because this was probably the next thing I was going to work on :P

**Comment by ValentijnvdBeek** [comment]
I wouldn't have recommended it since it requires a pretty big rethink on how Clay hotkeys work in general (which are kinda pain to be honest). 

The reason why 40928f99fe51b1aa628fd0e3f4fb5a0d7f674c1d doesn't include it is because I wanted to be able to easily reverse the commit it if our eternal god emperor @and3rson disagreed with it.
